blog/docs/articles/aws_to_func_migration.md

```diff
@@ -73,6 +73,13 @@ Note that this complete program contains only one function, which is focused on
 
 Looking closer at the signature of the `Handle` function, we see the standard `Context` API and an `Event` type. This is no vendor specific import. It references the Golang SDK for [CNCF CloudEvents](https://www.cncf.io/projects/cloudevents/){:target="_blank"}, which is a specification for describing event data in a common way.
 
+In this example the `subject` is mapped to the name of the file, or the S3 Object Key, while the `source` attribute is containing the bucket name. In case the entire file is desired for processing, it can be accessed via the `data` attribute.
+
+!!! note
+
+    The CNCF CloudEvents specification allows a generic and independent approach for receiving events from 3rd party systems, while providing a common, standardized API.
```

Comment:
Would it be worth to add link to spec here?

Reply:
we link to the project already in the document. and the AWS bits are not really spec'ed out. Apache Camel K was using "common sense" for the mapping :sweat_smile: